var/spack/repos/builtin/packages/genfit/package.py

```diff
@@ -29,14 +33,18 @@ class Genfit(CMakePackage):
     depends_on("eigen")
     depends_on("googletest")
 
+    # See https://github.com/GenFit/GenFit/pull/127
+    conflicts("root@6.30:", when="@:02-00-04", msg="genfit cannot be built against root@6.30 ")
+
     def cmake_args(self):
         args = []
-        # normally, as a cmake package root should be
-        # automatically picked up after 'depends_on'
-        # as it is added to CMAKE_PREFIX_PATH
-        # but genfit cooks its own root cmake config
-        # so this workaround is needed for now.
-        root_prefix = self.spec["root"].prefix
-        args.append("-DROOT_DIR=%s" % root_prefix)
+        if self.spec.satisfies("@:02-00-04"):
+            # normally, as a cmake package root should be
+            # automatically picked up after 'depends_on'
+            # as it is added to CMAKE_PREFIX_PATH
+            # but genfit cooks its own root cmake config
+            # so this workaround is needed for now.
+            root_prefix = self.spec["root"].prefix
+            args.append("-DROOT_DIR=%s" % root_prefix)
```

Comment:
Nit.  You could use a python f-string here (see https://docs.python.org/3/tutorial/inputoutput.html).